### PR TITLE
Improve accessibility and SEO

### PR DESCRIPTION
**Pull request by Danilo-Zekovic**

### CODE REVIEW

File: src/components/ContactForm/ContactForm.scss [hunk 1]
```diff
@@ -28,22 +28,22 @@
...
+  // }
 
-  button:active {
-    box-shadow: 1px 1px $black;
-    transform: translateY(2px);
-  }
+  // button:active {
+  //   box-shadow: 1px 1px $black;
+  //   transform: translateY(2px);
+  // }
```
Comment (vladislavaleksic88) [inline src/components/ContactForm/ContactForm.scss:48]: This is left commented after moving to a separate component.

File: src/components/Layout/Layout.tsx [hunk 2]
```diff
@@ -1,11 +1,33 @@
...
+          lang: 'en',
+        }}
+      >
+        <meta
+          name="description"
+          content="Official website of NeonCat software development company from Novi Sad, Serbia."
+        />
+        <meta
+          name="keywords"
+          content="NeonCat, Software, Development, Engineering, Solutions, Computer Science,  "
```
Comment (vladislavaleksic88) [inline src/components/Layout/Layout.tsx:27]: Comma and space left after the last keyword.

File: src/pages/index.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import Contact from '../components/Contact'
 
 const IndexPage: FC = () => {
   return (
-    <Layout>
+    <Layout title="Home">
```
Comment (vladislavaleksic88) [inline src/pages/index.tsx:10]: Do we need "Neon Cat - Home" in the title tag? Maybe we should leave just "Neon Cat" until we start the blog part?

Comment (Danilo-Zekovic) [reply src/pages/index.tsx:10]: I thought to add it right away as we will soon move to a blog page as well.

**Review by vladislavaleksic88 — APPROVED** [review]

**Pull request merged**